CDS/scripts/aspera.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+# This module uploads files to a server via Aspera.
+#
+# It assumes that you have Aspera installed and working.
+#
+#Arguments:
+# <host>sftp.hostname.com - upload to this server 
+# <username>blah - log in with this username 
+# <remote_path>/path/to/upload/ -  change to this directory to upload the file. Should end with a /.
+# <password>text - Password for the Aspera server.
+#END DOC
+
+if [ "${debug}" != "" ]; then
+echo
+set
+echo -----------------
+fi
+
+DATASTORE=`which cds_datastore.pl`
+if [ "${DATASTORE}" == "" ]; then
+	echo "-ERROR: Unable to find CDS datastore interface. Check that CDS is installed properly."
+	exit 1
+fi
+
+USERNAME=`${DATASTORE} subst "$username"`
+HOST=`${DATASTORE} subst "$hostname"`
+PASSWORD=`${DATASTORE} subst "$password"`
+if [ "$HOST" == "" ]; then HOST=$hostname; fi
```

Comment:
I'd exit with an error if hostname is not set; since the local machine is highly unlikely to be running an Aspera server this would just thow an ugly error if you forgot to include it in the route.  I'd rather have a nice error, like -ERROR: You need to specify <hostname> in the routefile